filters/cloudychat/cloudychat.go

```diff
@@ -25,7 +25,7 @@ var globalChatLimiter *rate.Limiter
 var log l4g.Logger
 var supervisor_log l4g.Logger
 var redis radix.Client
-var GameVersion = packets.ServerStatusVersion{"1.14.4", 498}
+var GameVersion = packets.ServerStatusVersion{"1.15.2", 578}
```

Comment:
Duplicate of https://github.com/Craftserve/potoq/pull/2/commits/13d7e30edd57b62ebd97fe9e6fb41388316e6ddd#diff-b7913550f26df0fe93932596ef6086c2R40

Reply:
To raczej powinno w packets/ byc gdzies z innymi stalymi moim zdaniem. Ten soft i tak nie obsluguje multiwersji aktualnie, to po co powtarzac to?